server/src/services/cfadock.service.ts

```diff
@@ -0,0 +1,25 @@
+import axios, { AxiosResponse } from "axios"
+
+const cfaDockEndpoint = "https://www.cfadock.fr/api/opcos"
+
+export const CFADOCK_FILTER_LIMIT = 100
+
+const getSirenParams = (sirenSet: Set<string>): any[] => {
```

Comment:
```suggestion
const getSirenParams = (sirenSet: Set<string>): string[] => {
```

ce n'est pas plutot un tableau de string qui est retourné ?

Reply:
non c'est des objets
```
{
  siret: "..."
}

```

Reply:
La fonction est inutile. Un Set peut être converti en Array de la manière suivante : 

```js
Array.from(Set) 
```
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/from